templates/org.eld

```diff
@@ -25,6 +25,12 @@ org-mode
  (format-time-string "%Y-%m-%d") n ":EXPORT_HUGO_DRAFT: false" n ":END:")
 (rdmecollapse  "*** " (p "Heading") n "#+HTML: <details> <summary> " (p "sub-heading")  " </summary>" n
  (r> "link or any comments") n n "#+HTML: </details>" n)
+(py_   & "#+begin_src python" n r n "#+end_src"
```

Comment:
Can you move it to the line 22? When we will have a consistency with other `_src` code blocks. This will be the last issue I see here

Reply:
Ok, I moved them to line 22, under GNU Plot.